# Shows if a list of joints is ready

string[] joints

---

bool is_ready
